src/fprime_gds/common/testing_fw/api.py

```diff
@@ -257,6 +277,78 @@ def wait_for_dataflow(self, count=1, channels=None, start=None, timeout=120):
             assert False, msg
         self.remove_telemetry_subhistory(history)
 
+    def get_config_file_path(self):
+        """
+        Accessor for IntegrationTestAPI's deployment configuration file.
+
+        Returns:
+            path to user-specified deployment configuration file (str)
+        """
+        if hasattr(self.pipeline, "deployment_config"):
+            return self.pipeline.deployment_config
+
+    def load_config_file(self):
+        """
+        Load user-specified deployment configuration JSON file.
+
+        Returns:
+            JSON object as a dictionary
+        """
+        config_file = self.get_config_file_path()
+
+        try:
+            with open(config_file, 'r') as file:
+                result = json.load(file)
+        except FileNotFoundError:
+            msg = f"Error: File not found at path {config_file}"
+            self.__log(msg, TestLogger.RED)
+            assert False, msg
+        except json.JSONDecodeError as e:
+            msg = f"Error decoding JSON: {e}"
+            self.__log(msg, TestLogger.RED)
+            assert False, msg
+        except Exception as e:
+            msg = f"An unexpected error occurred: {e}"
+            self.__log(msg, TestLogger.RED)
+            assert False, msg
+        return result
+
+    def getCmdDispName(self, instance=None, name=None):
+        """
+        Get deployment mnemonic of specified item from user-specified deployment
+        configuration file.
+
+        Args:
+            instance: native component to instance (str), i.e. "<component>.<instance>"
+            name: command, channel, or event name (str) [optional]
+        Returns:
+            deployment mnemonic of specified item (str) or native mnemonic (str) if not found
+        """
+        data = self.load_config_file()
+
+        if data:
+            try:
+                mnemonic = data[instance]
+            except KeyError:
+                self.__log(f"Error: {instance} not found", TestLogger.YELLOW)
+                return f"{instance}.{name}" if name else f"{instance}"
+            return f"{mnemonic}.{name}" if name else f"{mnemonic}"
+
+    def getPrmDbFilename(self):
+        """
+        Get file path to parameter db from user-specified deployment configuration file.
+
+        Returns:
+            file path to parameter db (str) or None if not found
+        """
+        data = self.load_config_file()
+
+        if data:
+            try:
+                return data["Svc.PrmDb.filepath"]
```

Comment:
Very nitpicky but is that the key name? 
It's a little misleading, one could think the JSON representation under the hood would be
```json
{
   "Svc" : { 
       "PrmDb": { "filepath" : "/abc/"}, 
        [...] 
    } 
}
```
But I'm assuming it's just
```json
{
    "Svc.PrmDb.filepath" : "/abc/:
}
```
Do we have an example of how a real-world config file would look like? It would be good to reference it here somewhere.